docs/use_cases/met_tool_wrapper/METdbLoad/METdbLoad.py

```diff
@@ -0,0 +1,116 @@
+"""
+METdbLoad: Basic Use Case
+=========================
+
+met_tool_wrapper/METdbLoad/METdbLoad.conf
+
+"""
+##############################################################################
+# Scientific Objective
+# --------------------
+#
+# Load MET data into a METviewer database using the met_db_load.py script
```

Comment:
Recommend that we replace the phrase "METviewer database" with something else since this db is used by both METviewer and METexpress:
```suggestion
# Load MET data into a METplus database using the met_db_load.py script
```

Reply:
How about just "Load MET data into a database..." ?

Reply:
Sure, "into a database" or "into METdatadb" is fine... just not a "METviewer database".